Update two topics to clarify current behavior (#4510)
segmentio/segment-docs: src/connections/sources/catalog/libraries/website/javascript/troubleshooting.md

## Is there a size limit on requests?

> Yes, 32KB per event message. Events with a payload larger than 32KB are accepted by Analytics.js with the browser getting responses `200` from our servers, but the event will be silently dropped once it enters our pipeline. 

Reviewer: ```suggestion
Yes, 32KB per event message. Events with a payload larger than 32KB are accepted by Analytics.js, with the browser returning a `200` response, but the event will be silently dropped once it enters Segment's pipeline. 
```

```
This works for any [context field](/docs/connections/spec/common/#context) that Segment automatically collects.

> When working with Page calls, you can overwrite context fields by following the same instructions above. However, because the `context.page` fields are also available in the `properties` parameter for page calls, you can also prevent the same fields in the `properties` parameter from being included in your Page call. The example below will allow you to overwrite `url` available in context field `page.url` and properties parameter:

Reviewer: ```suggestion
When working with Page calls, you can overwrite context fields by following the same instructions above. However, because the `context.page` fields are also available in the `properties` parameter for page calls, you must also prevent the same fields in the `properties` parameter from being included in your Page call. The example below will allow you to overwrite `url` available in context field `page.url` and properties parameter:
```
